app/api/service/auth/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask import jsonify, make_response, current_app
+from werkzeug.security import check_password_hash
+import jwt
+from app.models.models import User
+
+
+class AuthService:
+
+    @staticmethod
+    def login(data):
+
+        if not data or not data['username'] or not data['password']:
```

Comment:
are we guaranteed to have `username` and `password` keys in the data? if not, we're at risk of `KeyError`